### PR TITLE
[ZEPPELIN-2080] Fix null column parse issue in KylinInterpreter

### PR DESCRIPTION
**Pull request by yiming187**
### What is this PR for?
KylinInterpreter use regex pattern "\"results\":\\[\\[\"(.*?)\"]]" to extract the result, but if the last column is null, then no quotes in that field. The regex pattern will fail to match.
The KylinInterpreter should consider to support the null field case.

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-2080

### How should this be tested?
Has prepared the testcase in UnitTest

### Questions:
* Does the licenses files need update?No
* Is there breaking changes for older versions?No
* Does this needs documentation?No

### CODE REVIEW

File: kylin/src/main/java/org/apache/zeppelin/kylin/KylinInterpreter.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ private InterpreterResult executeQuery(String sql) throws IOException {
     return rett;
   }
 
-  private String formatResult(String msg) {
+  public String formatResult(String msg) {
```
Comment (jongyoul) [inline kylin/src/main/java/org/apache/zeppelin/kylin/KylinInterpreter.java]: LGTM except it. How about changing the package of test class as same as this class and make this method as package-local?

Comment (yiming187) [inline kylin/src/main/java/org/apache/zeppelin/kylin/KylinInterpreter.java]: Thank, jongyoul. Has updated the code.

**Comment by jongyoul** [comment]
Thanks. LGTM. If CI finished and become green, will merge it.

**Comment by felixcheung** [comment]
LGTM

**Pull request closed without merging**